packages/web/src/scss/foundation/_links.scss

```diff
@@ -7,11 +8,17 @@
 a {
```

Comment:
I would consider this
```
a {
    text-decoration-skip-ink: none;
    text-underline-offset: 0.1875em;
    color: tokens.$action-link-primary-default;

    &:active {
        text-decoration: underline;
        color: tokens.$action-link-primary-active;
    }

    @media (hover: hover) {
        &:hover {
            text-decoration: underline;
            color: tokens.$action-link-primary-hover;
        }
    }
}
```

this way we can avoid generating three rows of CSS instead of one in every selector we use the mixin. Everything is inherited. 

Also if we stick with the mixin, shouldn't it be placed in `tools/_links.scss`?

Reply:
also I expected that the stylelint would require the @include to be first 😨

Reply:
Thank you so much. I've totally missed this. It is fixed now.